app/src/main/java/co/nimblehq/alpine/sample/nfc/PassportInformationActivity.kt

```diff
@@ -0,0 +1,32 @@
+package co.nimblehq.alpine.sample.nfc
+
+import android.os.Bundle
+import android.view.inputmethod.EditorInfo
+import androidx.activity.ComponentActivity
+import co.nimblehq.alpine.databinding.ActivityPassportInformationBinding
+
+class PassportInformationActivity : ComponentActivity() {
```

Comment:
How about we update the name of the `Activity` to `MrzInfoActivity` since the user only submits the MRZ information on this screen?
```suggestion
class MrzInfoActivity : ComponentActivity() {
```
Need to update the layout file name as well.

Reply:
Yep, I will update 🚀